examples/streamlit/installer.cfg

```diff
@@ -0,0 +1,81 @@
+[Application]
+name=Windows Streamlit App
+version=0.1.0
+# How to lunch the app - this calls the 'main' function from the 'run_app' package:
+entry_point=src.run_app:main
+icon=resources/caritas-logo.ico
+
+[Python]
+version=3.8.10
+bitness=64
+
+[Include]
+# Packages from PyPI that your application requires, one per line
+# These must have wheels on PyPI:
+pypi_wheels = altair==4.1.0
+    astor==0.8.1
+    attrs==21.2.0
+    backcall==0.2.0
+    backports.zoneinfo==0.2.1
+    base58==2.1.0
+    bleach==4.1.0
+    blinker==1.4
+    cachetools==4.2.2
+    certifi==2021.5.30
+    cffi==1.14.6
+    charset-normalizer==2.0.6
+    click==7.1.2
+    decorator==5.1.0
+    defusedxml==0.7.1
+    distlib==0.3.3
+    entrypoints==0.3
+    idna==3.2
+    jsonschema==3.2.0
+    mistune==0.8.4
+    mypy-extensions==0.4.3
+    numpy==1.21.1
+    packaging==21.0
+    pandas==1.3.3
+    pandocfilters==1.5.0
+    parso==0.8.2
+    pillow==8.3.2
+    platformdirs==2.4.0
+    prompt-toolkit==3.0.20
+    protobuf==3.18.0
+    pyarrow==5.0.0
+    pycparser==2.20
+    pydeck==0.7.0
+    pyparsing==2.4.7
+    pyrsistent==0.18.0
+    python-dateutil==2.8.2
+    pytz==2021.1
+    requests==2.26.0
+    requests-download==0.1.2
+    send2trash==1.8.0
+    setuptools==57.0.0
+    six==1.14.0
+    smmap==4.0.0
+    streamlit==1.0.0
+    terminado==0.12.1
+    testpath==0.5.0
+    toml==0.10.2
+    tomli==1.2.1
+    toolz==0.11.1
+    tornado==6.1
+    traitlets==5.1.0
+    typing-extensions==3.10.0.2
+    tzlocal==3.0
+    urllib3==1.26.7
+    validators==0.18.2
+    # panda dependencies
+    Jinja2==3.0.1
+    # Jinja2 dependencies
+    MarkupSafe==2.0.1
+    # Plotly and its dependencies
+    plotly==5.3.1
+    tenacity==8.0.1
+    # fpdf
+    fpdf2==2.4.5
```

Comment:
I'd love to have a smaller list of packages for the example. It's a bit tricky, though: Streamlit has a huge amount of 'dependencies' which probably aren't really required - e.g. it depends on pydeck, which in turn depends on the whole Jupyter notebook. The long list you've got here still isn't everything that gets installed if you `pip install streamlit`.

https://github.com/streamlit/streamlit/issues/2703

I assume that pydeck is just one thing you can use in streamlit to display data.

I suspect that some more of these packages are probably unused - e.g. send2trash, which I know is a Jupyter dependency. But there's no easy way to find out which ones, so we may as well leave the list as is.

Reply:
I agree, I would have liked to have a smaller list too. I refused to put there the Jupyter dependencies, and fortunately it worked. I've already spent at least one day of work with trail and error to discover some missing dependencies, so I've just included almost everything. 

I notice now that I left here the fpdf2 dependency (last line), which indeed is needed for my specific app, so we may discard at least this one.

Reply:
Thanks :smiley: